scripts/config.pl

```diff
@@ -88,7 +89,7 @@
 MBEDTLS_PSA_CRYPTO_SPM
 MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER
 MBEDTLS_PSA_INJECT_ENTROPY
-MBEDTLS_USE_PSA_CRYPTO
```

Comment:
This wasn't introduced in a merge commit, but came from upstream mbedtls where it was done in a separate PR. https://github.com/ARMmbed/mbedtls/pull/2403/commits/971dea37453c9c45a64f32d91bf4e6da73d0c047

Reply:
This is being introduced as part of the merge of mbedtls, and we should not do it as part of a merge, especially since it would cause us to stop testing ECP_RESTARTABLE.

Reply:
The goal of a merge is to bring two branches together, aligning them, with the destination branch gaining all the applicable advantages and disadvantages of the source branch.

We should align on all.sh, but it's too bad Mbed TLS currently has this disadvantage.